agent/client/workflow_listen_for_ds_connection_tests.go

```diff
@@ -8,42 +8,55 @@ import (
 
 	"github.com/kubeshop/tracetest/agent/proto"
 	"github.com/kubeshop/tracetest/server/telemetry"
+	"go.uber.org/zap"
 )
 
 func (c *Client) startDataStoreConnectionTestListener(ctx context.Context) error {
+	logger := c.logger.Named("dataStoreConnectionTestListener")
+	logger.Debug("Starting")
+
 	client := proto.NewOrchestratorClient(c.conn)
 
 	stream, err := client.RegisterDataStoreConnectionTestAgent(ctx, c.sessionConfig.AgentIdentification)
 	if err != nil {
+		logger.Error("could not open agent stream", zap.Error(err))
 		return fmt.Errorf("could not open agent stream: %w", err)
 	}
 
 	go func() {
 		for {
 			req := proto.DataStoreConnectionTestRequest{}
 			err := stream.RecvMsg(&req)
+			if err != nil {
+				logger.Error("could not get message from data store connection stream", zap.Error(err))
+			}
 			if isEndOfFileError(err) || isCancelledError(err) {
+				logger.Debug("data store connection stream closed")
 				return
 			}
 
 			reconnected, err := c.handleDisconnectionError(err)
 			if reconnected {
+				logger.Debug("reconnected to data store connection stream")
```

Comment:
I think this one should be a warning instead of debug, especially because I want to know that the error was handled.

Reply:
fixed